core/sql/sqlcomp/nadefaults.cpp

```diff
@@ -2180,6 +2180,10 @@ SDDkwd__(ISO_MAPPING,           (char *)SQLCHARSETSTRING_ISO88591),
   XDDkwd__(MDAM_SCAN_METHOD,			"ON"),
 
   DDflt0_(MDAM_SELECTION_DEFAULT,		"0.5"),
+
+  DDflt0_(MDAM_TOTAL_UEC_CHECK_MIN_RC_THRESHOLD, "10000"),
+  DDflt0_(MDAM_TOTAL_UEC_CHECK_UEC_THRESHOLD,	 "0.01"),
```

Comment:
Thanks, Qifan, you make a good point. Sorry, my example isn't very convincing. Just wanted to run these thoughts by you, if you feel that 0.01 is the right number then I'm ok to go with it. Certainly it's a lot better than just counting the columns! Changing the number later will be relatively easy.